Experimental/DistributedAuthoritySample/Assets/Scripts/UI/IngameUI/UIUtils.cs

```diff
@@ -0,0 +1,72 @@
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Unity.Multiplayer.Samples.SocialHub.UI
+{
+    public static class UIUtils
+    {
+        public const string activeUSSClass = "show";
```

Comment:
consts should be named as k_PascalCase. This can be made internal too.

Reply:
its weird, i use the unity standards in rider and it underlines it if i have it k_
<img width="453" alt="Screenshot 2024-10-16 at 14 44 15" src="https://github.com/user-attachments/assets/ddcb7894-c783-4f3d-ae3f-ef9fe4ae3e85">

Reply:
done